website/blog/2025-12-05-kubernetes-ai-conformance-aks/index.md

```diff
@@ -0,0 +1,188 @@
+authors: [ahmed-sabbour]
+tags: [ai, best-practices, general]
+---
+
+As organizations increasingly move AI workloads into production, they need consistent and interoperable infrastructure they can rely on. The Cloud Native Computing Foundation (CNCF) launched the **Kubernetes AI Conformance Program** to address this need by creating open, community-defined standards for running AI workloads on Kubernetes.
+
+Azure Kubernetes Service (AKS) is proud to be among the first platforms certified for Kubernetes AI Conformance, demonstrating our commitment to providing customers with a verified, standardized platform for running AI workloads.
+
+<!-- truncate -->
+
+## What is the Kubernetes AI Conformance Program?
+
+The [Kubernetes AI Conformance Program](https://github.com/cncf/k8s-ai-conformance) defines a standard set of capabilities, APIs, and configurations that a Kubernetes cluster must offer to reliably and efficiently run AI and ML workloads. Building on CNCF's successful [Certified Kubernetes Conformance Program](https://www.cncf.io/certification/software-conformance/), which brought together more than 100 certified distributions and platforms, this new initiative applies the same proven model to AI infrastructure.
+
+The program is developed in the open by the [WG AI Conformance](https://github.com/kubernetes/community/tree/master/wg-ai-conformance) working group. This group operates under an openly published charter and focuses on:
+
+- Defining a reference architecture for AI workloads on Kubernetes
+- Establishing framework support requirements
+- Creating test criteria for key capabilities such as GPU integration, volume handling, and job-level networking
+
+## Why AI Conformance matters
+
+Running AI workloads on Kubernetes introduces unique challenges. Teams often struggle with GPU driver compatibility, specialized scheduling requirements for distributed training, and the complexity of exposing inference endpoints at scale. Without a common baseline, organizations risk building on fragmented, vendor-specific implementations that limit flexibility.
+
+According to [Linux Foundation Research on Sovereign AI](https://www.linuxfoundation.org/hubfs/Research%20Reports/lfr_sovereign_ai_090525a.pdf), 82% of organizations are already building custom AI solutions, and 58% use Kubernetes to support those workloads. With 90% of enterprises identifying open-source software as critical to their AI strategies, the risk of fragmentation and inconsistent performance is rising. The Kubernetes AI Conformance Program responds directly to this need.
+
+The AI Conformance Program addresses these challenges by establishing a verified set of capabilities that every conformant platform must support. When you deploy on AKS, a certified AI-conformant platform, you benefit from:
+
+- **Predictable scaling**: Your training jobs and inference services scale consistently using standardized APIs and autoscaling behaviors

```

Comment:
```suggestion
- **Predictable scaling**: Your AI workloads scale consistently using standardized APIs and autoscaling behaviors

```

Training jobs (not post training/RL jobs) usually do not require scaling and prefer static infra. Let's not say Training jobs.